bigbluebutton-html5/imports/ui/components/panel-manager/component.jsx

```diff
@@ -104,8 +104,8 @@ class PanelManager extends Component {
         enable={resizableEnableOptions}
         key={this.userlistKey}
         size={{ width: userlistWidth }}
+        onResize={() => window.dispatchEvent(new Event('resize'))}
```

Comment:
You could add to a function and call in each event listener, instead re-write the code.